From b965886a1bbf003693812f3871106fbe7efb33c9 Mon Sep 17 00:00:00 2001
From: Paul Harrison <pharrison@mythtv.org>
Date: Tue, 25 Mar 2014 16:40:13 +0000
Subject: [PATCH 1/2] Playlist: silence a failed query warning when inserting
 an empty playlist

Seems QSqlQuery treats QStrings that haven't been assigned to as NULL not an
empty string so 'QString rawList;' is not the same as 'QString rawList = "";'
as far as QSqlQuery is concerned.
---
 mythplugins/mythmusic/mythmusic/playlist.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mythplugins/mythmusic/mythmusic/playlist.cpp b/mythplugins/mythmusic/mythmusic/playlist.cpp
index b794ef1..79a509a 100644
--- a/mythplugins/mythmusic/mythmusic/playlist.cpp
+++ b/mythplugins/mythmusic/mythmusic/playlist.cpp
@@ -853,7 +853,7 @@ void Playlist::fillSonglistFromList(const QList<int> &songList,
 
 QString Playlist::toRawSonglist(bool shuffled, bool tracksOnly)
 {
-    QString rawList;
+    QString rawList = "";
 
     if (shuffled)
     {
-- 
1.7.10.2

